index.html

```diff
@@ -1738,88 +1738,66 @@
 Link this to [[[#guardians]]].
 </aside>
 
-## Harassment
+## Protecting users from abusive behaviour
+
+<div class="practice">
+  <p>
+    <span class="practicelab" id="abuse-reporting">
+      Systems that allow for communicating on the Web must provide an
+      effective capability to report abuse. [=User agents=] and [=sites=] must
+      take steps to protect their users from abusive behaviour, and abuse
+      mitigation must be considered when designing web platform features.
```

Comment:
This consolidation of principles applies different kinds of obligations to different actors. Designing reporting systems is quite different from abuse mitigation in the design of web platform features.

Reply:
I think we addressed.

Reply:
We split this principle into two in the meeting today, which I think fixes this comment.